readthedocs/templates/homepage.html

```diff
@@ -133,23 +133,39 @@ <h3>{% trans "Featured Projects" %}</h3>
   {% endif %}
   {% endcache %}
 
+  <section>
+    <h2>{% trans 'About Read the Docs' %}</h2>
+
+    <p>
+      Read the Docs has grown substantially
```

Comment:
Some of the copy here should have `trans`/`blocktrans` to be consistent.

Reply:
I added them, although I'm not 100 % sure of being consistent. @agjohnson would you please confirm?